### PR TITLE
Log errors as well when starting Librarian

### PR DESCRIPTION
**Pull request by paramaggarwal**
Currently errors get swallowed up and not logged causing difficulty in debugging.

### CODE REVIEW

File: setup.js [hunk 1]
```diff
@@ -143,6 +143,11 @@ const beginSetup = async (preferences) => {
     }
   });
 
+  bundler.stderr.on('data', (data) => {
+    log(chalk.red('Error:'));
+    log(String(data));
+  });
+            
   bundler.on('exit', function (code, signal) {
```
Comment (biocross) [inline setup.js]: Thank you for your MR @paramaggarwal!

There's this code already to print out errors if the server process exits with an error code. Is there a specific error you faced that this missed out?

Comment (paramaggarwal) [reply setup.js]: If `jekyll serve` throws an error, then it does not show it on console. Just says that error occurred.

**Comment by paramaggarwal** [comment]
I had made the change in the wrong file. Please check now.

If `jekyll serve` throws an error, then it does not show it on console. Just says that error occurred, hence an attempt to log those as well.

**Comment by biocross** [comment]
Looks good, merging now. Thank you!

**Pull request merged**